plugin-api.d.ts

```diff
@@ -863,6 +863,7 @@ type Trigger =
       readonly device: 'KEYBOARD' | 'XBOX_ONE' | 'PS4' | 'SWITCH_PRO' | 'UNKNOWN_CONTROLLER'
       readonly keyCodes: ReadonlyArray<number>
     }
+  | { readonly type: 'ON_VIDEO_HIT'; readonly videoHitTime: number }
```

Comment:
@jkatz-figma Should this be ON_MEDIA_HIT?

Reply:
Yup, same deal as the other PR -- plugin typings will be a fast follow. Thanks for calling it out here